cvlab_medical/med_image_loaders.py

```diff
@@ -0,0 +1,71 @@
+import nibabel as nib
```

Comment:
Make rebase of newest commit.
File name has been changed after refactoring and cleaning up.